javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,46 @@
+
+//Flight booking fullname function
+
+function getFullName(firstName, surname, useFormalName = true, gender = "") {
+    if (useFormalName) {
+        if (gender === "male") {
+            return "lord " + firstName + surname;
+        } else if (gender === "female") {
+            return "lady " + firstName + surname;
+        }
+    }
+            return firstName + surname;
+        }
+    
+
+    console.log(getFullName("Benjamin ", "Hughes", true,"male"))
+    console.log(getFullName("Benjamin ", "Hughes", false))
+    console.log(getFullName("Stefany"," Romero",true,"female"))
+
+
+//Event Application
+
+function getEventWeekday(daysFromToday){
+
+const weekDays= [,"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+const today = new Date().getDate();
```

Comment:
`new Date().getDate()` returns the day of the month (1-31), not the day of the week. 
Use `new Date().getDay()` to get the current day of the week (0-6).